drivers/sensor/bmp085/Kconfig

```diff
@@ -0,0 +1,35 @@
+# Bosch BMP085 pressure sensor
+
+# Copyright (c) 2024, Mario Paja
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BMP085
+	bool "Bosch BMP085 pressure sensor"
+	default y
+	depends on DT_HAS_BOSCH_BMP085_ENABLED
+	select I2C
+	help
+	  Enable driver for the Bosch BMP085 pressure sensor.
+
+if BMP085
+
+choice
+	prompt "Oversampling mode"
+	default BMP085_ULTRALOWPOWER
+	help
+	  Specify the oversampling mode to be used by the driver.
+
+config BMP085_ULTRALOWPOWER
```

Comment:
These might be better served being done in dts as st has done for various sensor configurations, imagine a board with multiple bmp085 instance where you'd like different modes for each. Non-blocking comment from me, but something that caught my eye after approving.

Reply:
I will have a look into it. Thank you for the input :)

Reply:
@teburd I think this would address your point :)

Reply:
yes, very nice